src/main/java/org/jabref/gui/preferences/PreferencesSearchHandler.java

```diff
@@ -0,0 +1,91 @@
+package org.jabref.gui.preferences;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javafx.beans.property.Property;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.control.Labeled;
+
+class PreferencesSearchHandler {
+
+    private final ObservableList<PrefsTab> preferenceTabs;
+    private final StringProperty searchText;
+    private final List<String> labelNames;
+    private final ObservableList<PrefsTab> filteredPreferenceTabs;
+    private final Property<ObservableList<PrefsTab>> filteredPreferenceTabsProperty;
```

Comment:
You can direclty use a `ListProperty filtered = new SimpleListProperty()`

Reply:
Thanks, I fixed it in the recent commit